docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
-
-1. The host loaded the enclave, which generated a remote attestation report.
-2. The client connected to the host and downloaded the attestation report.
-3. The client used the attestation report to verify that it was connected to the enclave it expected, and that the
-   enclave was running in the expected mode.
-4. The client encrypted the "reverse-me" string using the public key provided in the attestation.
-5. As part of the message to the enclave, the client also included a public key for the enclave to use when responding.
-6. The client sent the encrypted message to the host's REST API.
-7. The host passed the encrypted message to the enclave.
-8. The enclave decrypted the client's message, reversed the string, and encrypted the response using the client's
+
+You have run your first enclave successfully.
+
+The following sequence of events produces the above output:
+
+1. The host loads the enclave, which generates a remote attestation report.
+2. The client connects to the host and downloads the attestation report.
+3. The client uses the attestation report to verify that it is connected to the enclave it expected, and that the
+   enclave is running in the expected mode.
+4. The client encrypts the "reverse-me" string using the public key provided in the attestation.
+5. In the message to the enclave, the client also includes a public key for the enclave to use when responding.
+6. The client sends the encrypted message to the host's REST API.
+7. The host passes the encrypted message to the enclave.
+8. The enclave decrypts the client's message, reverses the string, and encrypts the response using the client's
    public key.
-9. The enclave passed the encrypted message to the host, was sent back to the client as an HTTP response to the same
+9. The enclave passes the encrypted message to the host, and is sent back to the client as an HTTP response to the same
    request.
 
-Congratulations on running your first enclave!
-
 ### Enclave Constraint
-The `--constraint` parameter of the client command-line interface defines the 
-properties of an acceptable enclave. This includes the signing key 
+The `--constraint` parameter of the client defines the properties of an acceptable enclave.
+This includes the signing key, which is also present in the host output in the previous section:
 ```
 S:0000000000000000000000000000000000000000000000000000000000000000
 ```
-which we saw in the host output in the previous
-section.
 
-Try rerunning the command with a different code signer:
+Try rerunning the command with a different signing key:
 ```
-"S:2222222222222222222222222222222222222222222222222222222222222222 PROD:1 SEC:INSECURE"
+java -jar client/build/libs/client.jar "S:2222222222222222222222222222222222222222222222222222222222222222 PROD:1
+SEC:INSECURE" reverse-me
 ```
 
-You should see the following error:
+You will get the following error, as the client's signing key(code signer) does not match the enclave's signing key:
```

Comment:
space before the opening parenthesis

Reply:
Removed this line as part of another review comment.